conjure-runtime/src/test.rs

```diff
@@ -479,106 +478,6 @@ failed-url-cooldown: 1h
     .await;
 }
 
-#[tokio::test]
```

Comment:
Since we're moving to more granular testing it seemed better to just remove these.